### DISCUSSION TITLE
Consider making %buildsystem_name_conf optional

### DISCUSSION THREAD

**Discussion by hroncok** [rpm-software-management/rpm#3191] [Ideas]
`%conf` is an optional section. Hence, I believe mandating the presence of `%buildsystem_name_conf` for a declarative buildsystem is suboptimal.

I was able to "solve" this by:

```
%buildsystem_name_conf() %nil
```

However, that seems rather boilerplate-like.

Is there a reason to force the presence of `%buildsystem_name_conf`?

**Comment by pmatilai** [comment]
Opened https://github.com/rpm-software-management/rpm/issues/3244 from this, along with a more elaborate answer why it's that way. Thanks for the suggestion.
